static/js/zamboni/reviewers.js

```diff
@@ -58,6 +58,37 @@
         window.location.hash = "id=" + $("#addon, #persona").attr("data-id");
       });
     }
+
+    // Toggle expansion for whiteboard
+    $("#id_whiteboard-public").click(function(){
+        $("#id_whiteboard-public").css("height", "200px");
+    });
```

Comment:
Your JS looks likes it would work (not tried locally yet) but does just using the `:active` css rule work instead?  It seems like we only want the whiteboard collapsed when it's empty and not active.

Reply:
I tried` :focus` css rule as well. It would do the collapsing work. However, by using `:focus` css rule, the whiteboard may still remain collapsed when there are contents on it but it is not selected. I am not very sure if remaining collapsed when there are contents on the whiteboard is what we want. If so, I could change it to use `:focus` instead.

FYI, the current JS and CSS solution will perform as follow (When the whiteboard does not have content and is not selected, it will collapse. If the whiteboard does have contents and is not selected, it will remain expanded.):
![more_action2](https://user-images.githubusercontent.com/46657851/77169719-5c9f4c80-6a90-11ea-992b-787a6c86a190.gif)

Reply:
A colleague suggested making use of the `:placeholder-shown` css rule instead to detect when the textarea is empty.

e.g. 
```
+        textarea:placeholder-shown {
+            height: 20px;
+        }
```

you'd just have to add define a placeholder in the `WhiteboardForm.meta` class with something like:
```
+       widgets = {
+            'private': widgets.Textarea(attrs={'placeholder': _('some placeholder text here')}),
+            'public': widgets.Textarea(attrs={'placeholder': _('some placeholder text here')}),
+        }
```

We'd rather not add extra javascript to the reviewer pages if possible - there's no test framework set up for it on addons-server so it's quite fragile.